### PR TITLE
Improve help message

### PR DESCRIPTION
**Pull request by xilopaint**
The redundant “ProtonVPN VPN” sounds weird to me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,11 +37,11 @@ $ sudo bash -c "git clone https://github.com/ProtonVPN/protonvpn-cli.git ; ./pro
...
-| `protonvpn-cli -c, --connect`                | Select a VPN from ProtonVPN menu.             |
-| `protonvpn-cli -c [server-name] [protocol]`  | Connect to a direct ProtonVPN VPN server.     |
-| `protonvpn-cli -r, --random-connect`         | Connect to a random ProtonVPN VPN.            |
-| `protonvpn-cli -f, --fastest-connect`        | Connect to a fast ProtonVPN VPN.              |
-| `protonvpn-cli -d, --disconnect`             | Disconnect from VPN.                          |
+| `protonvpn-cli -c, --connect`                | Select a server from the menu.                |
+| `protonvpn-cli -c [server-name] [protocol]`  | Connect directly to a ProtonVPN server.       |
+| `protonvpn-cli -r, --random-connect`         | Connect to a random ProtonVPN server.         |
+| `protonvpn-cli -f, --fastest-connect`        | Connect to a fast ProtonVPN server.           |
+| `protonvpn-cli -d, --disconnect`             | Disconnect from the ProtonVPN server.         |
```
Comment (Rafficer) [inline README.md]: I would maybe change that to `Disconnect the current session`.

IMO that makes it even clearer.

**Comment by Rafficer** [comment]
Why are there so many other changes that have nothing to do with the help message?

File: README.md (hunk 1 above)
Comment (xilopaint) [reply README.md]: Done.

**Comment by xilopaint** [comment]
Oh, sorry! I didn't figure out the other changes! I was working in this branch some days ago. I will revert the unrelated changes now.

**Comment by xilopaint** [comment]
@Rafficer is it possible to undo acac061 without reverting the other commits?

**Comment by Rafficer** [comment]
Should be, but I'm not too sure...

Maybe @mazen160 can chime in and answer (:

**Comment by xilopaint** [comment]
Now I've fixed the mess.

**Comment by Rafficer** [comment]
Not too sure... Did you remove the file completely and add it again?

Wouldn't this reset the history completely and you can't see what line is from which commit?

If so, then it might be better to close this PR and reopen a new one with a new branch and just the few lines of code edited ^^

**Comment by xilopaint** [comment]
I don't know if I get what you say. If you click on the "Files changed" tab you will see what lines were changed in the base branch.

**Comment by mazen160** [comment]
I will also make a new change on this based on a discussion today :)

**Comment by Rafficer** [comment]
Yeah, I'm dumb. Sorry ^^

**Comment by mazen160** [comment]
@Rafficer There is a cool option to merge everything as a single commit, which clears-up everything :)

**Comment by Rafficer** [comment]
@mazen160 Good to know, thanks! :)

**Comment by xilopaint** [comment]
> There is a cool option to merge everything as a single commit, which clears-up everything :)

Is there something link this for pull requests?

**Comment by mazen160** [comment]
@xilopaint I'm not sure, but it exists on merging pull requests. It's called "Squash & Merge"

**Pull request merged**